sdk/servicebus/service-bus/test/internal/unit/abortSignal.spec.ts

```diff
@@ -255,8 +280,6 @@ describe("AbortSignal", () => {
         assert.equal(err.message, StandardAbortMessage);
         assert.equal(err.name, "AbortError");
       }
```

Comment:
It was a bit redundant _but_ are the isLinkLocked calls being removed because the new cancellable lock doesn't support it?

Reply:
Yep exactly. We weren't using that API in any of our runtime code (or test code in event hubs) so I didn't implement it in the cancellable lock. We _could_ always add it if we thought we'd gain something from it though.